src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles.java

```diff
@@ -244,11 +243,6 @@ public void onTraversalStart() {
         if (outputType == CommonCode.OutputType.BQ) {
             loadStatus = new LoadStatus(projectID, datasetName, loadStatusTableName);
 
-            long streamingBufferRows = BigQueryUtils.getEstimatedRowsInStreamingBuffer(projectID, datasetName, loadStatusTableName);
```

Comment:
HI @kcibul  - we've had a little bit of a discussion about this. We're not quite sure what you were looking for here. Is it worth keeping this logging (and underlying method) around?